nestkernel/model_manager_impl.h

```diff
@@ -79,54 +81,59 @@ ModelManager::register_preconf_node_model( const Name& name,
   return register_node_model_( model, private_model );
 }
 
-template < typename ConnectionT, template < typename > class ConnectorModelT >
+// we're templating this function with a class template, e.g.
+// BernoulliConnection< targetidentifierT >
```

Comment:
Could you move this information to the Doxygen comment for the method in the h-file?